ababot/bot/src/commands/stonk.rs

```diff
@@ -1,12 +1,60 @@
+use crate::types::stonk::Stonk;
```

Comment:
This is out of scope for this PR, it adds a new command on top of the async change. Ok this time